source/Calamari.AzureAppService/AppSettingsManagement.cs

```diff
@@ -98,7 +98,7 @@ public static class AppSettingsManagement
 
             var results = await client.GetStringAsync(targetUrl);
 
-            var output = JsonConvert.DeserializeObject<appSettingNamesRoot>(results).properties.appSettingNames ??
+            var output = JsonConvert.DeserializeObject<appSettingNamesRoot>(results)?.properties.appSettingNames ??
```

Comment:
did this cause a problem when you were testing?

Reply:
NewtonSoft.Json returns a nullable string compared to System.Text.Json returning a string. I had to swap to NewtonSoft because of the Calamari.CloudAccounts net452 build